os/syscall/syscall_not_support.c

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ * Copyright 2025 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <errno.h>
+#include <stdint.h>
+#include <sys/types.h>
+#include <semaphore.h> 
```

Comment:
Why do we need the above 3 include files here? Are we using any declarations from these headers?